stable/anchore-engine/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: anchore-engine
-version: 0.1.7
-appVersion: 0.1.10
+version: 0.1.8
```

Comment:
I was going to wait to bump when we move to StatefulSets and remap how processes are run in the containers (now it's multiple procs per container, will move to one per container), but to track the app version I'm fine with a minor version bump if that is best practice.

Reply:
This is a significant enough change that its worth at least a minor. For the stateful set change you'll need to do a major version bump since it will almost certainly be backwards incompatible.